app/Http/Controllers/Admin/ColleaguesCrudController.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Backpack\CRUD\app\Http\Controllers\CrudController;
+
+// VALIDATION: change the requests to match your own file names if you need form validation
+use App\Http\Requests\AuthorsRequest as StoreRequest;
+use App\Http\Requests\AuthorsRequest as UpdateRequest;
+
+/**
+ * Class AuthorsCrudController
+ * @package App\Http\Controllers\Admin
+ * @property-read CrudPanel $crud
+ */
+class ColleaguesCrudController extends CrudController
+{
+    public function setup()
+    {
+        $this->crud->setModel('App\Models\Colleagues');
+        $this->crud->setRoute(config('backpack.base.route_prefix') . '/colleagues');
+        $this->crud->setEntityNameStrings('colleagues', 'colleagues');
+
+        $this->crud->addColumn([
+            'name'=>'name',
+            'label'=>'Name',
+            'type'=>'text',
+        ]);
+
+        $this->crud->addColumn([
+            'name'=>'jobs',
+            'label'=>'Jobs',
+            'type'=>'textarea',
+        ]);
+        
+        $this->crud->addColumn([
+            'name'=>'image',
+            'label'=>'Image',
+            'type'=>'image',
+            'prefix'=>'storage/authors_images/'
+        ]);
+
+        $this->crud->addColumn([
+            'name'=>'category',
+            'label'=>'Category',
+            'type' => 'select_from_array',
+            'options' => [0 => 'Brass', 1 => 'Teachers', 2=>'lecturer', 3=>'Administrator', 4=>'Kitchen Staff', 5=>'Maintenance Staff'],
+            'allows_null' => false,
+        ]);
+        
+        $this->crud->addField([
+            'name'=>'image',
+            'label'=>'Profile image',
+            'type'=>'image',
+            'prefix'=>'storage/authors_images/'
```

Comment:
This full field is bad:
 - Your not uploading the file to anywhere
 - The prefix you `set` points to the wrong disk
 - The prefix does not need a '/' at the end, in fact it makes the resulting URL invalid

Reply:
Fixed uploading.